### DISCUSSION TITLE
Using specific commits of the Webui/Dreambooth in a Colab Notebook

### DISCUSSION THREAD

**Discussion by NickJack4** [d8ahazard/sd_dreambooth_extension#1043] [Q&A]
I'm very new to this, so I apologize in advance if this is a dumb question or if the solution isn't worth the trouble. I'm wondering how difficult it would be to set up a Colab notebook that pairs prior commits of the WebUI (7ba3923) and Dreambooth (65a4bdf). While the newer version is more customizable and some experts have gotten fantastic results, I found a specific pairing of the WebUI and DB that I've actually been able to use to isolate simple variables that I understand and can learn from via trial and error.

Unfortunately, my hardware struggles to get two or three models trained in my free time, so I'm not able to do much experimentation with my local installation that I set up using git checkout. That's why I wanted to try it with Colab, but I keep getting errors.

Thanks

**Comment by G-force78** [comment]
Yeah a collab of the latest version would be great. Im going to try dropping the train-dreambooth file from this repo into shivams collab and see what happens.
